### ISSUE TITLE
Error when PORT Environment Variable is Defined

### ISSUE THREAD

**Issue by Californian** [prisma/prisma#2795]
# Bug description
When the `PORT` environment variable is set, the Prisma CLI fails with a cryptic error (below). My assumption is that this is due to the `--port` flag for Prisma Studio.
`"prisma" Error: Get config error: Invalid value for '--port <port>': cannot parse integer from empty string`

## How to reproduce
1. `export PORT=8000`
2. `npx prisma generate`

## Expected behavior
At worst, a clearer error should be exposed. Ideally, the environment variable should be disambiguated (just as my API's port environment variable should be, which resolves the problem for my current purposes).

## Environment & setup
- OS: tested on Alpine, Ubuntu and Arch
- Database: PostgreSQL
- Prisma version: `prisma/1.34.10 (linux-x64) node-v12.18.0`
- Node.js version: erbium

**Comment by janpio** [comment]
Is it possible that you do not have a Prisma 2.x installation in the folder you are trying to execute this? That would download Prisma 1, which is currently available as the Npm package `prisma`.

**Comment by Californian** [comment]
No, when I unset the variable it works as intended.

**Comment by Californian** [comment]
It's possible I posted the wrong Prisma version here though now that I
think of it. I installed 2.0.1 but then just ran npx outside of that when
writing the issue because I didn't think it would be different and it was
faster than spinning up that environment.

**Comment by janpio** [comment]
This is what it looks like with Prisma 2 answering the command:

```
C:\Users\Jan                                                                                   
λ set PORT=9999                                                                                
                                                                                               
C:\Users\Jan                                                                                   
λ prisma generate                                                                              
                                                                                               
                                                                                               
info You don't have defined any generator in your schema.prisma, so nothing will be generated. 
You can define them like this:                                                                 
                                                                                               
generator client {                                                                             
  provider = "prisma-client-js"                                                                
}                                                                                                                                                                                          
```

**Comment by janpio** [comment]
This is what it looks like with Prisma 2 answering the command:

```
C:\Users\Jan\Documents\throwaway\2795  (2795@1.0.0)                                                 
λ npm install @prisma/cli                                                                           
                                                                                                    
> @prisma/cli@2.0.1 preinstall C:\Users\Jan\Documents\throwaway\2795\node_modules\@prisma\cli       
> node preinstall/index.js                                                                          
                                                                                                    
                                                                                                    
> @prisma/cli@2.0.1 install C:\Users\Jan\Documents\throwaway\2795\node_modules\@prisma\cli          
> node download-build/index.js                                                                      
                                                                                                    
npm notice created a lockfile as package-lock.json. You should commit this file.                    
npm WARN 2795@1.0.0 No description                                                                  
npm WARN 2795@1.0.0 No repository field.                                                            
                                                                                                    
+ @prisma/cli@2.0.1                                                                                 
added 1 package from 1 contributor and audited 1 package in 6.985s                                  
found 0 vulnerabilities                                                                             
                                                                                                    
                                                                                                    
C:\Users\Jan\Documents\throwaway\2795  (2795@1.0.0)                                                 
λ set PORT=999                                                                                      
                                                                                                    
C:\Users\Jan\Documents\throwaway\2795  (2795@1.0.0)                                                 
λ npx prisma generate                                                                               
Error: Either provide --schema or make sure that you are in a folder with a schema.prisma file.     
```

Interesting enough I still get a different error message after uninstalling Prisma 2:

```
C:\Users\Jan\Documents\throwaway\2795  (2795@1.0.0)                                             
λ npm uninstall -g @prisma/cli                                                                  
removed 1 package in 0.311s                                                                     
                                                                                     
C:\Users\Jan\Documents\throwaway\2795  (2795@1.0.0)                                             
λ npx prisma generate                                                                           
npx: installed 594 in 42.408s                                                                   
 !    Couldn’t find `prisma.yml` file. Are you in the right directory?                          
                                                                                                
Get in touch if you need help: https://slack.prisma.io                                          
To get more detailed output, run $ export DEBUG="*"                                             
(node:252) [DEP0066] DeprecationWarning: OutgoingMessage.prototype._headers is deprecated       
```

Same if I change the ENV var to an empty string:
```                                                                                            
C:\Users\Jan\Documents\throwaway\2795  (2795@1.0.0)                                             
λ set PORT=""                                                                                   
                                                                                                
C:\Users\Jan\Documents\throwaway\2795  (2795@1.0.0)                                             
λ npx prisma generate                                                                           
npx: installed 594 in 32.059s                                                                   
 !    Couldn’t find `prisma.yml` file. Are you in the right directory?                          
                                                                                                
Get in touch if you need help: https://slack.prisma.io                                          
To get more detailed output, run $ export DEBUG="*"                                             
(node:19084) [DEP0066] DeprecationWarning: OutgoingMessage.prototype._headers is deprecated     
```

**Comment by matthewmueller** [comment]
A lot has changed since June. Please re-open if you're still facing this @Californian!

**Comment by keyworks** [comment]
FYI I experienced the same issue with `prisma generate` : 
```
$ pnpx prisma generate
Prisma schema loaded from prisma/schema.prisma
Error: Get config error: Invalid value for '--port <port>': cannot parse integer from empty string
```
I unset the PORT variable and it appears to get past the issue. My @prisma/client is on 2.15.0. I'm on node 10.22.0 and ubuntu 18.

Maybe this should be re-opened.

**Comment by pantharshit00** [comment]
@keyworks This is fixed in dev channel and will be released in 2.16: https://github.com/prisma/prisma/issues/4789

We eliminated our usage of this variable completely.
